src/server/models/parking-lot.ts

```diff
@@ -0,0 +1,30 @@
+import { Document, Schema, model, Model } from 'mongoose';
+
+export interface ParkingLot extends Document {
+  name: string;
+}
+
+export const ParkingLotSchema = new Schema<
+  ParkingLot,
+  Model<ParkingLot, ParkingLot>
+>(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+  },
+  {
+    versionKey: false,
+  },
+);
+
+ParkingLotSchema.methods.toJSON = function (): Thruway.ParkingLot {
+  return {
+    name: this.name,
+  };
+};
```

Comment:
Let have `[Model prototype].toJSON` methods include the ObjectID, what do you think?

Reply:
Yeah, we must to return the ObjectID for use later with other entities.